ext/tags.go

```diff
@@ -22,6 +22,9 @@ var (
 
 	// PeerPort records port number of the peer
 	PeerPort = uint16Tag("peer.port")
+
+	// SamplingPriority determines the priority of sampling this Span.
+	SamplingPriority = uint32Tag("sampling.priority")
```

Comment:
You mean values are converted to string by the Span?  First, it's the implementation detail of this particular tracer, some tracers may want to store values in the native format without conversion. But more importantly, the point here is to provide type safety, for example you won't be able to set a port tag with a string garbage 
```
PeerPort.Add(span, "garbage")
```
the compiler will require `uint16` argument to this `Add` method.